app/views/components/_document-list.html.erb

```diff
@@ -0,0 +1,50 @@
+<%
+  group_index ||= false
+  track_clicks = "data-module=track-click" if group_index
+  dimension28 = items.count.to_s
+%>
+<ol class="app-c-document-list" <%= track_clicks %>>
```

Comment:
It should be an OL, yes. Documents are provided from the content store in alphabetical order, which means pairs of documents occur in the right order (see the 'Assessment' section of https://www.gov.uk/government/collections/statutory-guidance-schools)